### DISCUSSION TITLE
建议右上角历史图标展开每个视频后面都有一个删除

### DISCUSSION THREAD

**Discussion by Drismer001** [BewlyBewly/BewlyBewly#1172] [Ideas]
建议主页右上角历史图标展开每个视频后面都有一个删除，节省点开历史视频界面再删除视频
![屏幕截图 2024-12-07 134820](https://github.com/user-attachments/assets/e40ab37f-6347-4ee3-a97a-d7ab2846c724)

**Comment by hakadao** [comment]
已記錄 https://github.com/BewlyBewly/BewlyBewly/issues/1178
